notebooks/api/0.8/01-submit-code.ipynb

```diff
@@ -328,6 +329,16 @@
     "project = projects[0]"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Or when working on a project that already exists\n",
+    "project = guest_domain_client.get_project(name=\"My Cool UN Project\")"
```

Comment:
It would good also to add assert statements, as pytest stops the execution only with exception(Link in the case where a project does not exist).on that particular line itself
I have added it in my upcoming commit.

Pinging here, as it would be helpful in the upcoming PR's